### PR TITLE
CRW-1462 - Upgrade VS Code Language Support for Apache Camel to 0.0.30

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

File: dependencies/che-plugin-registry/v3/plugins/redhat/vscode-apache-camel/0.0.30/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 title: Language Support for Apache Camel
 description: This VS Code extension provides support for Apache Camel.
 icon: https://raw.githubusercontent.com/redhat-developer/codeready-workspaces/master/dependencies/che-plugin-registry/resources/images/default.svg?sanitize=true
 repository: https://github.com/camel-tooling/camel-lsp-client-vscode
 category: Language
-firstPublicationDate: '2020-12-04'
+firstPublicationDate: '2021-01-05'
 spec:
   containers:
   - image: "registry.redhat.io/codeready-workspaces/plugin-java11-rhel8:2.5"
```
Comment (ericwill) [inline dependencies/che-plugin-registry/v3/plugins/redhat/vscode-apache-camel/0.0.30/meta.yaml:15]: This should probably be updated to `registry.redhat.io/codeready-workspaces/plugin-java11-rhel8:2.6`

cc @nickboldt

Comment (nickboldt) [reply dependencies/che-plugin-registry/v3/plugins/redhat/vscode-apache-camel/0.0.30/meta.yaml:15]: correct, always use latest.

Comment (apupier) [reply dependencies/che-plugin-registry/v3/plugins/redhat/vscode-apache-camel/0.0.30/meta.yaml:15]: why is it not updated when there is a version upgrade?

**Comment by apupier** [comment]
@ericwill @nickboldt Can you review it please? (to avoid me a potential 3rd rebase and conflict resolution)

**Review by ericwill — APPROVED** [review]
Sorry, forgot about this one. LGTM.

**Pull request merged**